### ISSUE TITLE
screen reader activating regardless of state in settings>accessibility>enable screen reader>off

### ISSUE THREAD

**Issue by f3bandit** [ValveSoftware/SteamOS#2011]
#### Your system information

* Steam client version: 1750726487
* SteamOS version: 3.7.12
* Opted into Steam client beta?: yes 
* Opted into SteamOS beta?: yes
* Have you checked for updates in Settings > System?: yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen.

#### Steps for reproducing this issue:

Decky loader installed and using the magic black plugin while downloading updates while plugged in and charging.
Screen reader is activating only when  turning on magic black plugin. Screen reader then starts reading the screen while the screen is off. Magic black has a button combo of quick access button and the view button. Screen reader uses  steam button and the view button. accessibility features are disabled in settings. yet screen reader keeps activating when using the magic black button combo. 
 
1.  Activate Magic Black with quick access and view buttons
2.  Screen goes black as expected, but screen reader activates as if the steam and view buttons were pressed
3.  check settings and screen reader still states de-activated
4.  Activate Magic Black with quick access and view buttons
5.  Screen goes black as expected, but screen reader activates as if the steam and view buttons were pressed
yet again.
